ivi-layermanagement-api/test/TestBase.cpp

```diff
@@ -78,10 +78,25 @@ shm_format(void *data, struct wl_shm *wl_shm, uint32_t format)
     base->SetShmFormats(format);
 }
 
-static struct wl_shm_listener shm_listener = {
-    shm_format
+struct shm_listener_t {
```

Comment:
why to use struct in this case? It is a class clearly which all the relevant methods in place.

Reply:
I agree. I updated this to class in newest commit